app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -51,12 +51,21 @@ public function panel(Panel $panel): Panel
                 'warning' => Color::Amber,
                 'blurple' => Color::hex('#5865F2'),
             ])
+            ->renderHook(
+                'panels::sidebar.footer',
+                fn () => view('filament.Footer'),
+            )
+            ->navigationItems([
+                NavigationItem::make('client')
+                    ->label('Return to client')
+                    ->url('/') // TODO once the Filament client side is done this can be changed to /client
```

Comment:
```suggestion
                    ->url(route('route.name.here'))
```

Reply:
I tried this at first and it did not work for some reason i get this error:

![image](https://github.com/pelican-dev/panel/assets/62171904/f4865cb1-2b3e-48d2-a02b-44067c96ee04)

So i did this. It works just as well